src/coreclr/jit/codegencommon.cpp

```diff
@@ -10899,9 +10780,9 @@ void CodeGen::genIPmappingGen()
             if ((block->bbRefs > 1) && (stmt != nullptr))
             {
                 bool found = false;
-                if (stmt->GetILOffsetX() != BAD_IL_OFFSET)
+                if (stmt->GetDebugInfo() != BAD_IL_OFFSET)
```

Comment:
Yep, looks like this code is under `#if 0`, I'll update the block so that it compiles when it is enabled. It looks like something I may wish to try to enable in the future.